### ISSUE TITLE
Modify build for Pi Zero 2

### ISSUE THREAD

**Issue by elevins** [geaz/simplyRetro-Z5#19]
Any tips on how to approach this?

**Comment by geaz** [comment]
As far as I know, the Pi Zero 2 is same size as the Pi Zero. Should be a drop in replacement.
But the provided os image will probably not work with the Pi Zero 2.

**Comment by elevins** [comment]
Yeah, I used it as a drop-in replacement and the image would not boot. Saw this issue https://github.com/geaz/simplyRetro-Z5/issues/18 and it seems to be the culprit. 

Since I've already wired everything up and the gpio used are not compatible with retropie or recalbox it looks like i have two choices. 

1) rewire the gpio and use one of the other os designed for running retro games
2) recompile the os for use with the rpi2 zero 

Wondering if you had any docs or a tutorial you used to setup the embedded linux os. so i could try number 2 before i resort to path 1.

**Comment by geaz** [comment]
Your best bet is the official buildroot documentation: https://buildroot.org/
Its quit great and was a huge help the time I created the image.

As a third option you could also take retropie, for example, and just compile the "retroPower" application and use it within retropie for the battery indicator/power circuit.

**Comment by elevins** [comment]
Thanks will attempt to use buildroot and recompile the provided source files for the zero 2. Think the issue may be the upgrade to the new cpu for the rpi zero 2. Will update/close the issue with my findings.

**Comment by seanie5011** [comment]
> Thanks will attempt to use buildroot and recompile the provided source files for the zero 2. Think the issue may be the upgrade to the new cpu for the rpi zero 2. Will update/close the issue with my findings.

Hi, did you ever find a conclusion to this?
